addons/owncloud/models.py

```diff
@@ -129,7 +129,7 @@ def create_waterbutler_log(self, auth, action, metadata):
                 'project': self.owner.parent_id,
                 'node': self.owner._id,
                 'folder': self.folder_id,
-                'path': metadata['materialized'].strip('/'),
+                'path': metadata['materialized'],
```

Comment:
Minor: This should probably still `lstrip('/')`, if the original intent was aesthetics.